### ISSUE TITLE
The "How GuidedTrack Works" template seems out-of-place conceptually

### ISSUE THREAD

**Issue by lyudmil** [classicrob/GuidedTrack-Templates#2]
The rest of the templates are meant to teach the user how to accomplish specific tasks. "How GuidedTrack Works" is clearly meant to introduce the platform as a whole. It doesn't substantially differ from "Interactive Lesson Template (easy)" in terms of the features it's showing off.

The problem I see is that if the user chooses "How GuidedTrack Works" first, it would have a lot of value. It would have significantly less value if the user chooses another template first.

Would it make more sense to pull "How GuidedTrack Works" out of the templates one could use to create a program, and instead bake it into GuidedTrack so that the user sees this code first before having the option to create another program from a template?

**Comment by classicrob** [comment]
Yes, I agree! Users should be linked to "How GuidedTrack Works" directly from the onboarding questionnaire, at which point if they want to go back to the programs page and select a template, they can. 

That should have been clarified: this program, as well as the onboarding questionnaire and pre-signup-flow are not meant to be considered as templates. I just included them in the repo because they are programs we'll be using in the onboarding.
